test/builtin_dataset_mocks.py

```diff
@@ -1431,3 +1431,48 @@ def stanford_cars(info, root, config):
         make_tar(root, "car_devkit.tgz", devkit, compression="gz")
 
     return num_samples
+
+
+@register_mock
+def sun397(info, root, config):
+    images_root = root / "SUN397"
+    images_root.mkdir()
```

Comment:
No need for `os.makedirs()` here. In general, `pathlib` supports everything you need with paths. `.mkdir()` also takes the `exist_ok: bool` and `parents: bool` flags for more complex situations.